### PR TITLE
Homework 3

### PR DESCRIPTION
**Pull request by PashaMisan**
AWS IAM practice

### CODE REVIEW

File: awscli/policies/get_update_env_script.json
Comment (saaverdo) [inline awscli/policies/get_update_env_script.json:1]: Nice solution.
As an alternative, separate buckets for DB and APP routines could be created.

File: awscli/policies/read_ssm_db_params.json
Comment (saaverdo) [inline awscli/policies/read_ssm_db_params.json:1]: Dedicated tree for 'db' params - good!

**Review by saaverdo — APPROVED** [review]
Good work! :+1: 
Approved!)

File: awscli/policies/get_update_env_script.json
Comment (PashaMisan) [reply awscli/policies/get_update_env_script.json:1]: Sure thing, I'll keep that in mind for the future, thanks!

**Pull request merged**